packages/components/src/components/dropdown/dropdown.tsx

```diff
@@ -434,7 +436,12 @@ const Dropdown = ({
                 data-testid={test_id}
                 value={value || 0}
             />
-            <div ref={wrapper_ref} className={containerClassName()}>
+            <div
+                ref={wrapper_ref}
+                className={containerClassName()}
+                onMouseOver={should_open_on_hover ? () => setIsListVisible(true) : undefined}
```

Comment:
This functionality was added for showing dropdown `onHover` (if `should_open_on_hover` flag is `true`), not only `onClick`